kubetest/kubernetes.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os/exec"
+	"time"
+)
+
+// kubectlGetNodes lists nodes by executing kubectl get nodes, parsing the output into a nodeList object
+func kubectlGetNodes() (*nodeList, error) {
+	o, err := output(exec.Command("kubectl", "get", "nodes", "-ojson"))
```

Comment:
just a question: do you care which kubectl you use here? (from gcloud or from k8s tarball)